db/migrations/20231102052648_blacklist_misc_changes.php

```diff
@@ -0,0 +1,29 @@
+<?php
+declare(strict_types=1);
+
+use Phinx\Migration\AbstractMigration;
+
+final class BlacklistMiscChanges extends AbstractMigration
+{
+    public function up(): void {
+        $this->execute("
+            ALTER TABLE `chat` 
+                ADD `user_id` INT(11) NOT NULL DEFAULT 0 AFTER `post_id`,
+                ADD `deleted` SMALLINT(2) NOT NULL DEFAULT 0 AFTER `edited`;
```

Comment:
We need to add an index on `deleted` as well since it will be in the `WHERE` clause of every chat refresh query